include/xgboost/data.h

```diff
@@ -461,6 +461,17 @@ class DMatrix {
 #endif
                        const std::string& file_format = "auto",
                        const size_t page_size = kPageSize);
+
+  static DMatrix* LoadMultiple(std::vector<const std::string>& uris,
```

Comment:
Should we overload the `Load` function instead of using a new name `LoadMultiple`?

Reply:
Good idea. Done